pipeline/config/exoscale/sc-config.yaml

```diff
@@ -48,3 +48,8 @@ s3Exporter:
   scrapeTimeout: 30s
 prometheus:
   predictLinearLimit: 66
+prometheusBlackboxExporter:
+  targets:
+    fluentd: false
+    thanosQuery: false
+    prometheus: false
```

Comment:
If the pipeline doesn't differ from the default config you shouldn't need to add any of these